src/pages/page-2.js

```diff
@@ -142,7 +142,7 @@ import educationIcon from "../images/education_icon.png"
                 })}
                 <button className="btn btn--add btn--small" onClick={addWorkExp}><img src={workIcon} /> Werkervaring toevoegen</button>
 
-                <h1>Opleidingen {Math.round(height)}px tall</h1>
+                <h1>Opleidingen</h1>
```

Comment:
Ik zie dat we momenteel Engels en Nederlands door elkaar gebruiken. Met de komst van onze nieuwe collega denk ik dat het beter is om overal Engels te gebruiken :)

Reply:
Laat het voor nu staan, dat zou misschien beter een andere ticket kunnen zijn. Kunnen dan ook een toggle functie toevoegen.